src/Calculator/Package.appxmanifest

```diff
@@ -36,6 +36,17 @@
                         <uap:Logo>Assets\CalculatorAppList.png</uap:Logo>
                     </uap:Protocol>
                 </uap:Extension>
+                <uap3:Extension Category="windows.appExtension">
+                    <uap3:AppExtension Name="com.microsoft.windows.dontmaximizeonsmallscreen"
+                                       Id="calculator"
+                                       DisplayName="calculator"
+                                       Description="This app extension makes calculator no longer maximizing by default on small screen device."
```

Comment:
nit: Let's say "This app extension prevents calculator from being maximized by default on some small screen devices."

In both versions of the manifest.